package.json

```diff
@@ -489,6 +489,28 @@
           "group": "2_files",
           "when": "view == remoteExplorer && viewItem != root"
         },
+        {
+          "command": "sftp.upload.folder",
+          "group": "3_trans@1",
+          "alt": "sftp.forceUpload",
+          "when": "sftp.enabled && viewItem != file"
+        },
+        {
+          "command": "sftp.upload.file",
+          "group": "3_trans@1",
+          "when": "view == remoteExplorer && viewItem != root && viewItem == file"
+        },
+        {
+          "command": "sftp.download.folder",
+          "group": "3_trans@2",
+          "alt": "sftp.forceDownload",
+          "when": "sftp.enabled && viewItem != file"
+        },
+        {
+          "command": "sftp.download.file",
+          "group": "3_trans@2",
+          "when": "view == remoteExplorer && viewItem != root && viewItem == file"

```

Comment:
@NyaPPuu,
Please report the "_comment" section to the  `Download Folder` command.

Should look like this :
```json
        [...]
        {
          "command": "sftp.download.file",
          "group": "3_trans@2",
          "when": "view == remoteExplorer && viewItem != root && viewItem == file"
          "_comment": "!custom"
        }
        [...]
```

Reply:
@Natizyskunk 
What does the "_comment" section do?
Looks like a comment literally.

I tested the removed version and the added version, but there seems to be no difference in operation.
If you still want to add it, I will.

Reply:
You're actually right, apparently it does nothing and it also isn't used anywhere else.
My bad. You can leave it like taht without the `"_comment"` part.